pages/blog/release-notes-2.2.0.md

```diff
@@ -0,0 +1,62 @@
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.1.0'` into `asyncapi: '2.2.0'`
+
+## Assigning channels to servers
+
+This feature allows the definition of AsyncAPI documents for applications with more selective use of channels and servers. An example is message protocol adapters that consume messages from one server (say, Kafka) and publish those messages to another server (say, Anypoint MQ).
+
+Now you can add a new **servers** property to [Channel Item Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#channel-item-object). It must be a list of server names provided as a string.
+
+```yaml
+description: This application publishes WebUICommand messages to an AMQP queue on RabbitMQ brokers in the Staging and Production environments.
+servers:
+  - rabbitmqBrokerInProd
+  - rabbitmqBrokerInStaging
+subscribe:
+  message:
+    $ref: "#/components/messages/WebUICommand"
+bindings:
+  amqp:
+    is: queue
+```
+
+Names of servers must match the names of the servers defined in the [Servers Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#serversObject). This new property is optional, so moving from 2.1.0 to 2.2.0 is as easy as changing the specification version in your current AsyncAPI file. Suppose **servers** is absent or empty. In that case, the given channel must be available on all servers defined in the Servers Object, as it was before the 2.2.0 version.
+
+For more details, check out [this pull request](https://github.com/asyncapi/spec/pull/531).
+
+We heard some community members asking for this feature. It was [Gerald Loeffler](https://www.linkedin.com/in/geraldloeffler/) that decided to champion the proposal and lead it until it got released. Thank you :pray:.
+
+## New protocol bindings
+
+The specification is now extended to support the following custom protocols through the bindings feature:
+- **Anypoint MQ**, thanks to [Gerald Loeffler](https://www.linkedin.com/in/geraldloeffler/). For more details check out [this pull request](https://github.com/asyncapi/spec/pull/545) and [binding definition](https://github.com/asyncapi/bindings/tree/master/anypointmq).
+
+## Become a contributor
+
+Pushing things through into the specification is not an easy process are requires a lot of time and patience, but it is worth it. Have a look at our [contribution guide](https://github.com/asyncapi/spec/blob/master/CONTRIBUTING.md) and start contributing.
+
+## Conclusions
+
+Are you wondering how we managed to release 2.2.0 release just three months after 2.1.0 release? I recommend you familiarize yourself with [AsyncAPI release process](https://github.com/asyncapi/spec/blob/master/RELEASE_PROCESS.md). Brace yourself for the next release scheduled for January 2022.
+
+<img className="w-3/4" src="/img/posts/release-notes-2.2.0/brace.webp" />
```

Comment:
I think 3.0.0 will probably come out in April, not January :) We still don't have conclusions about receive/send proposition and many others, and let's not forget that 3.0.0 will involve breaking changes in parser, generator, templates, react-component etc, so a lot of work. Generally we should not give hope to people that 3.0.0 will come out in January and this meme points to it 😅

Reply:
Meme doesn't say anything about January. Also its point is not to give hopes, but "brace yourself", so be aware that it is coming, and there is really no other way, a workaround to not do it

Reply:
maybe I just modify sentence `Brace yourself for the next release scheduled for January 2022.` to `The next release scheduled for January 2022. The next releases are later in April, June and September.`

Reply:
there is no way we release 3.0 in January 😄

Reply:
I just pushed improvement, you should be happy

Reply:
Yeah, this second sentence is better :)

Reply:
Actually, you have a typo. Should be `The next release is scheduled for January 2022`. Would it make sense to involve @alequetzalli this time? I'm more than sure we will learn a lot about wording release notes, at least I will 😅

Reply:
well, grammar improvements, definitely 🙏🏼 . Change in wording and style?  no plans really, I sign it with my name ❤️ , happy to see the next release someone else doing it 😄 `release coordination` is a role anyone can take up 😉

Reply:
hahahaha _"release coordination is a role anyone can take up"_ 😂

I love helping w/ release notes because it helps me really learn in better detail the many items being shipped around me. If you want me to switch off and on with you, I am happy to help out! Or if you like doing it just fine and think you would feel good about a 2nd pair of eyes for quick grammar review or thoughts of improvements, I guess just let me know?! 😀😂😁😆

Reply:
I'm definitely up for a grammar review. Switching of roles just for release notes writing, not sure as release coordination is complex already. We might consider it for the next release. For this one, all is ready, no new things are coming in.